// then-busboy/main.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "RequestEntityTooLargeError", {
  enumerable: true,
  get: function () {
    return _RequestEntityTooLargeError.default;
  }
});
Object.defineProperty(exports, "FileSizeLimitError", {
  enumerable: true,
  get: function () {
    return _FileSizeLimitError.default;
  }
});
Object.defineProperty(exports, "FieldsLimitError", {
  enumerable: true,
  get: function () {
    return _FieldsLimitError.default;
  }
});
Object.defineProperty(exports, "FilesLimitError", {
  enumerable: true,
  get: function () {
    return _FilesLimitError.default;
  }
});
Object.defineProperty(exports, "PartsLimitError", {
  enumerable: true,
  get: function () {
    return _PartsLimitError.default;
  }
});
Object.defineProperty(exports, "parse", {
  enumerable: true,
  get: function () {
    return _parse.default;
  }
});
Object.defineProperty(exports, "default", {
  enumerable: true,
  get: function () {
    return _parse.default;
  }
});
Object.defineProperty(exports, "Body", {
  enumerable: true,
  get: function () {
    return _Body.default;
  }
});
Object.defineProperty(exports, "File", {
  enumerable: true,
  get: function () {
    return _File.default;
  }
});
Object.defineProperty(exports, "isFile", {
  enumerable: true,
  get: function () {
    return _File.isFile;
  }
});

var _RequestEntityTooLargeError = _interopRequireDefault(require("./lib/error/RequestEntityTooLargeError"));

var _FileSizeLimitError = _interopRequireDefault(require("./lib/error/FileSizeLimitError"));

var _FieldsLimitError = _interopRequireDefault(require("./lib/error/FieldsLimitError"));

var _FilesLimitError = _interopRequireDefault(require("./lib/error/FilesLimitError"));

var _PartsLimitError = _interopRequireDefault(require("./lib/error/PartsLimitError"));

var _parse = _interopRequireDefault(require("./lib/parse"));

var _Body = _interopRequireDefault(require("./lib/Body"));

var _File = _interopRequireWildcard(require("./lib/File"));

function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) { var desc = Object.defineProperty && Object.getOwnPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : {}; if (desc.get || desc.set) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } } newObj.default = obj; return newObj; } }

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
